content/manual/manual_circuit.rst

```diff
@@ -4,29 +4,30 @@ Circuit Construction
 
 .. Open DAG; equivalence up to trivial commutations/topological orderings
 
-The :py:class:`Circuit` class forms the unit of computation that we can send off to a quantum co-processor. Each instruction is to be performed in order, potentially parallelising when they use disjoint sets of (qu)bits. To capture this freedom of parallelisation, we treat the circuit as a Directed Acyclic Graph with a vertex for each instruction and directed edges following the paths of resources (e.g. qubits and bits) between them. This DAG representation describes the abstract circuit ignoring these trivial commutations/parallel instructions.
+A quantum circuit acts as the foundation for creating quantum algorithms. Within the `pytket` library, the :py:class:`Circuit` class serves as the fundamental component for constructing computations that are executable on a quantum co-processor. Each instruction within the circuit is executed sequentially, with the possibility of running instructions in parallel if they operate on separate sets of qubits and (classical) bits. To utilize the potential for parallel execution, the circuit is depicted as a Directed Acyclic Graph (DAG), with each operation forming a node and the movement of qubits and bits illustrated by connecting arrows (directed edges). This DAG layout provides an abstract view of the circuit, effectively managing the nuances of operation reordering and parallel actions.
```

Comment:
Perhaps change "depicted as a Directed Acyclic Graph (DAG)" to "represented internally as a Directed Acyclic Graph (DAG)"

Reply:
done